modules/consent/www/getconsent.php

```diff
@@ -139,16 +183,101 @@
     break;
 }
 
-if (array_key_exists('consent:store', $state)) {
-    $t->data['usestorage'] = true;
-} else {
-    $t->data['usestorage'] = false;
-}
+$t->data['usestorage'] = array_key_exists('consent:store', $state);
 
 if (array_key_exists('consent:hiddenAttributes', $state)) {
     $t->data['hiddenAttributes'] = $state['consent:hiddenAttributes'];
 } else {
     $t->data['hiddenAttributes'] = array();
 }
 
+$t->data['attributes_html'] = present_attributes($t);
+
 $t->show();
+
+/**
+ * Recursive attribute array listing function
+ *
+ * @param SimpleSAML_XHTML_Template $t          Template object
+ *
+ * @return string HTML representation of the attributes
+ */
+function present_attributes($t)
```

Comment:
We already have this function defined in `templates/includes/attributes.php`. Wouldn't it be better to just use that one?